arbeitszeit_flask/templates/macros/tooltips.html

```diff
@@ -0,0 +1,23 @@
+{% macro info_tooltip(id, text) %}
+<div class="dropdown is-right">
+    <div class="dropdown-trigger">
+        <span class="icon" aria-haspopup="true" aria-controls="{{ id }}">
+            <i class="fa-solid fa-info-circle">
+                <span class="icon is-small">
+                    <i class="fas fa-angle-down" aria-hidden="true"></i>
+                </span>
+            </i>
+        </span>
+    </div>
+    <div class="dropdown-menu" id="{{ id }}" role="menu">
+        <div class="dropdown-content">
+            <div class="dropdown-item has-text-left">
```

Comment:
I find the text a bit too small - what about adding the class "is-size-6"?

Reply:
done